src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,69 @@
+package duke.task;
+
+import java.util.LinkedList;
+
+/**
+ * Represents a task list. A <code>task list</code> object stores
+ * the information of all tasks in the program. It has two
+ * fields, which are the task list stored in the linked list and the number of the tasks.
+ */
+public class TaskList {
+    private LinkedList<Task> Tasks;
```

Comment:
Another curious question: why LinkedList and not perhaps ArrayList?

Reply:
I think both of them are OK.

Reply:
If you ask Duke to delete a certain task, say delete 3, then linked list might perform better than array list. 
But when you add a task to the end of the list, then array list performs better.

Reply:
I see where you're coming from now 👍